xbmc/addons/addoninfo/AddonInfo.h

```diff
@@ -140,6 +140,14 @@ class CAddonInfo
    */
   bool HasType(TYPE type, bool mainOnly = false) const;
 
+  /**
+   * @brief To check an addon's main type
+   *
+   * @param[in] type Type identifier to be checked
+   * @return true in case the wanted type is supported, false if not
+   */
+  bool IsMainType(TYPE type) const;
```

Comment:
"Is" reads strange. Why not "Has"? And addon **has** a type, but it **is** not a type, right?

Reply:
And the doc speaks about "supported", yet another verb, btw. ;-)

Reply:
yes, that's both correct :) .. but beside the naming of that function i'm sorry to say i've no idea why this test keeps failing

Reply:
@ksooo let's just call it `HasMainType()` - only the test is still a mystery to me. can't be right to just kick it out. what do you think?

Reply:
HasMainType would be fine for me